app/controllers/dependencies_controller.rb

```diff
@@ -29,41 +19,24 @@ def edit
   def update
     @dependency = Dependency.find(params[:id])
 
-    respond_to do |format|
-      if @dependency.update_attributes(dependency_params)
-        format.html { redirect_to @dependency, :notice => 'Dependency was successfully updated.' }
-        format.json { head :ok }
-      else
-        format.html {
-          render :action => "edit"
-        }
-        format.json { render :json => @dependency.errors, :status => :unprocessable_entity }
-      end
+    if @dependency.update_attributes(dependency_params)
```

Comment:
Yes, we do. That was removed to simplify the action since maybe we don't really use json requests, improving then the code climate score. If you plan to use those requests maybe it is worth to create a private method to avoid duplication, where possible